en/StopTimeUpdate.md

```diff
@@ -0,0 +1,12 @@
+### StopTimeUpdate
+
+| Field Name | Recommendation |
+| --- | --- |
+| stop_sequence | If `stop_sequence` and `stop_id` are both provided, any combination of `stop_sequence` and `stop_id` must have a match in GTFS data. |

```

Comment:
Should language containing "must" strictly belong in the spec instead?

Reply:
Yes, good catch. In best practices these should be "should"s. If we feel strongly enough that it should be "must", they should be proposed for adoption into the spec.

Reply:
I fixed this one in https://github.com/MobilityData/GTFS_Realtime_Best-Practices/pull/1/commits/3a91318684468283ce4f87d36831679da8435ad7.